Formula/tarantool@2.1.rb

```diff
@@ -0,0 +1,62 @@
+class TarantoolAT21 < Formula
```

Comment:
This is not needed

Reply:
But we need to add additional class for the additional 1.10.4 LTS version, is any other way exists to add it not it the tarantool@1.10.rb formula ?

Reply:
Please make a separate pull request to add any old versions.

Reply:
Created the new separate PRs.